realm/src/main/java/io/realm/internal/SharedGroup.java

```diff
@@ -40,9 +40,9 @@
         MEM_ONLY(1);
         //ASYNC(2); // TODO: re-enable when possible
 
-        final int value;
+        public final int value;
```

Comment:
I leave the enum for this PR. And If needed, I would like to create another PR to change all the internal enums to constants.